tools/StaticAnalysis/Exceptions/Az.Monitor/SignatureIssues.csv

```diff
@@ -7,3 +7,6 @@
 "Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.ScheduledQueryRules.NewScheduledQueryRuleScheduleCommand","New-AzScheduledQueryRuleSchedule","1","8100","New-AzScheduledQueryRuleSchedule Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.ScheduledQueryRules.NewScheduledQueryRuleSourceCommand","New-AzScheduledQueryRuleSource","1","8100","New-AzScheduledQueryRuleSource Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.ScheduledQueryRules.NewScheduledQueryRuleTriggerConditionCommand","New-AzScheduledQueryRuleTriggerCondition","1","8100","New-AzScheduledQueryRuleTriggerCondition Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.ScheduledQueryRules.NewScheduledQueryRuleTriggerConditionCommand","New-AzScheduledQueryRuleTriggerCondition","1","8100","New-AzScheduledQueryRuleTriggerCondition Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.Alerts.NewAzureRmMetricAlertRuleV2CriteriaCommand","New-AzMetricAlertRuleV2Criteria","1","8410","Parameter NumberOfViolations of cmdlet New-AzMetricAlertRuleV2Criteria does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
```

Comment:
These suppressions match the comments above.  You must remove them and fix the underlying issue.